Sprint #19.adoc

```diff
@@ -29,9 +29,45 @@
 * TASK: make hasAttachment searchable via mailbox-api
 * TASK: make GetMessagesList call the search api
 
+Remark. If you want to call here the search API for Messages you will have to problems :
+ - It returns a list of UID. You may need to add information about mailboxes depending on your implementations choices
+ - As such the returned document is not present (reduce network load) + the index document is not the original mail (text extraction). So you will still need a read to Cassandra after your read to ElasticSearch..
+
 == Questions
 
 * Do we want to manage quota on upload ? Ask that with INBOX and UX team
+
+From the spec : 
+
+....
+If uploading a file would take the user over quota, the server SHOULD delete previously uploaded (but unused) files before their expiry time. 
+....
+
+I guess so.
```

Comment:
First, it's a SHOULD
Second, it's not about what the spec says but what we want to implement first.